packages/core/src/endowmentsToolkit.js

```diff
@@ -295,17 +379,26 @@ function endowmentsToolkit({
       }
     }
     if (sourcePropDesc.set) {
+      const setter = sourcePropDesc.set
       wrappedPropDesc.set = function (value) {
         // replace the "receiver" value if it points to fake parent
+        // eslint-disable-next-line @typescript-eslint/no-this-alias
         const receiver = this
         const receiverRef =
           receiver === unwrapFromGlobalThis ? unwrapToGlobalThis : receiver
-        return Reflect.apply(sourcePropDesc.set, receiverRef, [value])
+        return Reflect.apply(setter, receiverRef, [value])
```

Comment:
(must) same issue as with https://github.com/LavaMoat/LavaMoat/pull/732/files#diff-5ed0581fdb5b3ec8105f7f2e6773d6f5db45dbe74dccf2a921cc9c0e40ecda60R354

Sorry I didn't point it out here. I did in a different PR that was showing it. 

GH will not let me add a suggestion that fixes it...

Reply:
done